app/src/main/java/de/westnordost/streetcomplete/data/osmnotes/CreateNoteDao.java

```diff
@@ -38,6 +41,10 @@ public boolean add(CreateNote note)
 		{
 			values.put(CreateNoteTable.Columns.ELEMENT_ID, note.elementId);
 		}
+		if (note.imagePaths != null)
+		{
+			values.put(CreateNoteTable.Columns.IMAGE_PATHS, note.imagePaths.toString());
```

Comment:
**Serialization issue:** You need to actually serialize the array list here, not toString() it.

Reply:
Done